### DISCUSSION TITLE
Support for older macOS versions

### DISCUSSION THREAD

**Discussion by latvia234** [vincode-io/Zavala#121] [General]
When I go to the App Store on my Mac I can see that I need at least macOS 11.0.

Do you know if Zavala can support older macOS versions? I'm using 10.14.6.

**Comment by vincode-io** [comment]
Unfortunately we can't support older versions. We are very dependent on [Mac Catalyst](https://developer.apple.com/mac-catalyst/) and specifically features that were only introduced in macOS 11.

Obviously I would have liked to make Zavala available to as many people as possible, but the work to support older versions of macOS is just too much for one person to do. Basically, I would have to completely write a new version of Zavala for macOS which wouldn't share nearly as much code with the iOS version. Ultimately, this would mean fewer features for both platforms.

I'm going to try to continue to support macOS 11 in version 2 of Zavala, but I can't commit to that. There are features like Shortcuts which are specific to macOS 12 that I really want to implement in v2. I'm just not sure yet how to support Shortcuts on macOS 12 and still deploy to macOS 11. I haven't looked into it much yet, but I hope Apple has provided a way to do so.
